tools/generate-docs-integration-screenshot

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python3
+"""Create or update a webhook integration screenshot using a test fixture."""
+
+# check for the venv
+from lib import sanity_check
+
+sanity_check.check_venv(__file__)
+
+import os
+import sys
+
+TOOLS_DIR = os.path.abspath(os.path.dirname(__file__))
+ROOT_DIR = os.path.dirname(TOOLS_DIR)
+
+sys.path.insert(0, ROOT_DIR)
+from scripts.lib.setup_path import setup_path
+
+setup_path()
+
+os.environ["DJANGO_SETTINGS_MODULE"] = "zproject.settings"
+import django
+
+django.setup()
+
+import argparse
+import subprocess
+from typing import Any, Dict, Tuple
+
+import cairosvg
+import requests
+import ujson
+
+from zerver.models import UserProfile, Message
+from zerver.lib.actions import do_create_user, notify_created_bot
+from zerver.lib.upload import upload_avatar_image
+from zerver.lib.actions import do_change_avatar_fields
+from zerver.lib.integrations import WebhookIntegration, INTEGRATIONS
+from zerver.lib.webhooks.common import get_fixture_http_headers
+
+def create_integration_bot(integration_name: str) -> UserProfile:
+    owner = UserProfile.objects.get(id=10)
+    bot_email = "{}-bot@example.com".format(integration_name)
+    bot_name = "{} Bot".format(integration_name.capitalize())
+    try:
+        bot = UserProfile.objects.get(email=bot_email)
+    except UserProfile.DoesNotExist:
+        bot = do_create_user(
+            email=bot_email,
+            password="123",
+            realm=owner.realm,
+            full_name=bot_name,
+            short_name=bot_name,
+            bot_type=UserProfile.INCOMING_WEBHOOK_BOT,
+            bot_owner=owner,
+        )
+        notify_created_bot(bot)
+
+        if integration.logo_url is None:
+            return bot
+        logo_path = os.path.join(ROOT_DIR, integration.logo_url.lstrip("/"))
+
+        if logo_path.endswith(".svg"):
+            logo_png_path = "/tmp/{}-logo.png".format(integration_name)
+            cairosvg.svg2png(url=logo_path, write_to=logo_png_path)
```

Comment:
(zulip-py3-venv) tabbott@coset:~/zulip$ ./tools/generate-docs-integration-screenshot zerver/webhooks/github/fixtures/pull_request__assigned.json 
Traceback (most recent call last):
  File "./tools/generate-docs-integration-screenshot", line 45, in create_integration_bot
    bot = UserProfile.objects.get(email=bot_email)
  File "/srv/zulip-py3-venv/lib/python3.6/site-packages/django/db/models/manager.py", line 82, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/srv/zulip-py3-venv/lib/python3.6/site-packages/django/db/models/query.py", line 408, in get
    self.model._meta.object_name
zerver.models.DoesNotExist: UserProfile matching query does not exist.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.6/urllib/request.py", line 1474, in open_local_file
    stats = os.stat(localfile)
FileNotFoundError: [Errno 2] No such file or directory: '/home/tabbott/zulip/http:/localhost:9991/static/images/integrations/logos/github.svg'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "./tools/generate-docs-integration-screenshot", line 105, in <module>
    bot = create_integration_bot(integration.name)
  File "./tools/generate-docs-integration-screenshot", line 64, in create_integration_bot
    cairosvg.svg2png(url=logo_path, write_to=logo_png_path)
  File "/srv/zulip-py3-venv/lib/python3.6/site-packages/cairosvg/__init__.py", line 70, in svg2png
    output_height=output_height)
  File "/srv/zulip-py3-venv/lib/python3.6/site-packages/cairosvg/surface.py", line 144, in convert
    **kwargs)
  File "/srv/zulip-py3-venv/lib/python3.6/site-packages/cairosvg/parser.py", line 402, in __init__
    parse_url(self.url), 'image/svg+xml')
  File "/srv/zulip-py3-venv/lib/python3.6/site-packages/cairosvg/parser.py", line 258, in fetch_url
    return read_url(url, self.url_fetcher, resource_type)
  File "/srv/zulip-py3-venv/lib/python3.6/site-packages/cairosvg/url.py", line 159, in read_url
    return url_fetcher(url, resource_type)
  File "/srv/zulip-py3-venv/lib/python3.6/site-packages/cairosvg/url.py", line 100, in fetch
This line seems to require the development server running; does it support either `file://` URLs or just passing a path instead of doing a `urlopen`?    One gets this exception is the server isn't running.

  ```
  return urlopen(Request(url, headers=HTTP_HEADERS)).read()
  File "/usr/lib/python3.6/urllib/request.py", line 223, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.6/urllib/request.py", line 526, in open
    response = self._open(req, data)
  File "/usr/lib/python3.6/urllib/request.py", line 544, in _open
    '_open', req)
  File "/usr/lib/python3.6/urllib/request.py", line 504, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.6/urllib/request.py", line 1452, in file_open
    return self.open_local_file(req)
  File "/usr/lib/python3.6/urllib/request.py", line 1491, in open_local_file
    raise URLError(exp)
urllib.error.URLError: <urlopen error [Errno 2] No such file or directory: '/home/tabbott/zulip/http:/localhost:9991/static/images/integrations/logos/github.svg'>
```

Reply:
Looks like 99242138a725477cd44c09b1069ce8b3d9c21aec seems to have broken this (which I hadn't pulled, while working locally). Sorry for the trouble!